src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -113,29 +113,34 @@ Here's more detail about the two authentication domain sections:
 
 ### Managing users [#managing-users]
 
-For how users are added and managed, you can select **Manually** or **Identity provider**. 
+For how users are added and managed, you can select **Manually** or **Identity provider**. The option to use your identity provider to provision users is available only if your organization has [Pro or Enterprise edition](https://newrelic.com/pricing). 
 
-The option to use your identity provider to provision users is available only if your organization has [Pro or Enterprise edition](https://newrelic.com/pricing). If you choose **Identity provider**, you must follow [additional steps for using specific identity providers](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). Once you do that, these users are created on the new user model and synced in New Relic based on your identity provider configuration. 
+If you choose **Identity provider**, **you must follow the [steps for automated user management](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign#how-to)**. Once you completely those steps return to the user migration wizard and docs. 
 
-Note that if you're already using an app made by Okta, Azure, or OneLogin, you'll probably need to use a newer version made for New Relic's newer user model. For more on this, see [Identity provider apps](#idp-apps).
+Some tips and recommendations for doing this step: 
 
-After you complete provisioning users, you can confirm that these users have been added by seeing if they’re in the new [**User management** UI](#manage-users).
+* If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date one. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."
```

Comment:
```suggestion
* If you're already using a New Relic app for either Okta, Azure, or OneLogin, you're likely using an out-of-date version. The out-of-date apps are titled "New Relic by account" while the newer apps are titled "New Relic by organization."
```

Reply:
Or, well. I suggested "version," but any other word that fits should be fine. I think "one" is a little vague and there might exist a stronger noun somewhere out there.